### DISCUSSION TITLE
GITHUB EDUCATION

### DISCUSSION THREAD

**Discussion by johndavelausa** [community/community#134965] [GitHub Education]
### Hi everyone,

Why i still didn't have GitHub Education? It's been 3 days and I haven't received it yet.

**Comment by davevad93** [comment]
Hi @xczqb ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @xczqb was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
